### DISCUSSION TITLE
New Diaper Sprites

### DISCUSSION THREAD

**Discussion by Gooberology** [zippity21/Stardew_Valley_Regression_Mod#88] [Show and tell]
Ive been working on some new diaper sprites for the pull request im going to release. Im not much of a pixel artist, but i am a diaper artist, and wanted to update the sprites to have a more subtle, realistic look to the usage. Im curious as to what you guys think, I think they could still use some touchups, but theyre nearly there. I resprited the cloth diaper from scratch to look more like a folded clothespin diaper, and changed the baby print diaper to have some pastel marks on it, the original i could only assume had a image of a baby on it?

![image2](https://user-images.githubusercontent.com/131845784/234478876-89011579-1599-44bd-9d80-b7850a284f89.png)

**Comment by Gooberology** [comment]
Should note that the undies are unchanged, since theres nothing to actually absorb or contain for them, i think its alright. For messing, i emphasized more on diaper droopage and a slight, dingy color. The print diapers were the hardest to look passable, I think the usage colors on white look really nice, but having to shift those usage colors to different hues was really difficult for me

**Reply by zippity21** [reply]
Ah, collaborative art. I think these look great!

 I'm thinking that as more sprite sheets are made, we should collect them in a common area so the user can choose, instead of having them scour the web. I think we're a small enough community that keeping track of them wouldn't be too cumbersome.

Eventually, I think we're going to need some more assets if we ever get around to implementing #10 .

**Reply by Gooberology** [reply]
Could just keep all the different sprite packs in the assets folder, and they just have to rename whatever they want to use as "sprites.png"
